server/log_rpc_server_test.go

```diff
@@ -2063,7 +2080,7 @@ func (p *parameterizedTest) executeCommitFailsTest(t *testing.T, logID int64) {
 	}
 
 	registry := extension.Registry{
-		AdminStorage: fakeAdminStorage(p.ctrl, storageParams{logID, p.preordered, 1, nil, nil}),
+		AdminStorage: fakeAdminStorage(p.ctrl, storageParams{logID, p.preordered, 1, nil, nil, false}),
```

Comment:
OK for now, but if there's more test parameters to come it might be worth shifting to `storageParams{treeID: logID, preordered:p.preordered, numSnapshots:1}` style initialization.

Reply:
There was a mixture of styles before I think. Not planning any more changes atm. Trying to wrap all this up today.